main.go

```diff
@@ -41,12 +41,27 @@ func mainError() error {
 	dnsServers := flag.String("dns-servers", "", "Colon separated list of DNS servers.")
 	hostname := flag.String("hostname", "", "Hostname of the tenant node.")
 	mainConfig := flag.String("main-config", "", "Path to main ignition config (appended to small).")
+	ntpServers := flag.String("ntp-servers", "", "Colon separated list of NTP servers.")
 	out := flag.String("out", "", "Path to save resulting ignition config.")
 	flag.Parse()
 
-	dnsServersList := strings.Split(*dnsServers, dnsServersDelimiter)
-	if len(dnsServersList) == 0 {
```

Comment:
This check didn't work because there was always one element in that list. See https://play.golang.org/p/zeYqKode51w